conn_test.go

```diff
@@ -385,10 +385,15 @@ func TestEmptyQuery(t *testing.T) {
 	db := openTestConn(t)
 	defer db.Close()
 
-	_, err := db.Exec("")
+	res, err := db.Exec("")
 	if err != nil {
 		t.Fatal(err)
 	}
+	if ra, err := res.RowsAffected(); err == nil {
+		t.Fatal("expected error")
+	} else if ra != 0 {
```

Comment:
Feels odd to look at "ra" if err != nil; per convention in Go that usually means "ra" must be ignored.  But see my comment about RowsAffected above.